containers/Ideas/IdeaCreate.js

```diff
@@ -38,8 +39,7 @@ export const IdeaCreate = props => {
       .required('Idea title is Required'),
     description: yup
       .string()
-      .min(20, ({ min }) => `Description must be at least ${min} characters`)
-      .max(100, 'Description must be no longer then 100 characters')
+      .max(10000, 'Description must be no longer then 100 characters')
```

Comment:
I guess this is still needed in this view to show, validation error while not in the description component?
10000 feels quite a lot of chars though.

just saw the text says 100, but the actual limit is 10000

Reply:
this i left there till we knew what limit should be but there actually no limit so will remove